### DISCUSSION TITLE
Undo doesn't work on "remove segment"

### DISCUSSION THREAD

**Discussion by PeterJerde** [mifi/lossless-cut#2130] [General]
When a segment is selected, hitting the delete key on the keyboard or pressing the "-" button below the segments removes it. But then "undo" does not bring it back! Am I doing something wrong, or is that a bug? ◡̈

**Comment by mifi** [comment]
Hi! are you using the top menu undo action or keyboard shortcut to undo?

Undo/redo segments doesn't work through the top menu. This is a [known issue](https://github.com/mifi/lossless-cut/issues/610) that I don't know how to fix. Please use the keyboard shortcuts instead (<kbd>CTRL</kbd>/<kbd>CMD</kbd>+<kbd>Z</kbd> and <kbd>CTRL</kbd>+<kbd>Y</kbd> / <kbd>CMD</kbd>+<kbd>SHIFT</kbd>+<kbd>Z</kbd>).

**Reply by Galaxoid-333** [reply]
I don't understand programming, but if possible, please explain why the top menu items cannot call the keyboard command Ctrl + Z , which will undo. And the keys Ctrl + Y for redo also do not work.

**Reply by mifi** [reply]
it's a limitation in Electron. and i haven't found a workaround. see the linked issue above.
